networks/local.json

```diff
@@ -1,7 +1,7 @@
 {
   "network": "mainnet",
-  "accessKey": {
-    "address": "0x44b6317365fb117c540dccfd29348f94d5bbffdc",
-    "startBlock": 0
+  "AppFactory": {
+    "startBlock": 3,
```

Comment:
Why is the `startBlock` 3 here? Can it go back to `0`?

Reply:
I took the start block from contracts deployment run-latest.json
Can set it back to 0 yes